src/nodes/edge.cpp

```diff
@@ -133,3 +164,16 @@ NodeConstants::ProcessResult Edge::pull()
 
     return NodeConstants::ProcessResult::Unchanged;
 }
+
+// Express as a serialisable value
+SerialisedValue Edge::serialise() const { return definition().serialise(); }
+
+// Read values from a serialisable value This is required for the
+// SerialableValue type implementation, but we actually deserialise
+// Edges through an EdgeConnection.  I've added this error to
+// immediately alert us in case this function is ever called.
+void Edge::deserialise(const SerialisedValue &node)
+{
+    throw std::runtime_error("Cannot directly deserialise edges.  Please contact the Dissolve development team if you are "
+                             "seeing this error - this is a bug and NOT your fault.");
```

Comment:
Just to ensure the poor recipient of the error sees the message!
```suggestion
                             "seeing this error - this is a bug and NOT your fault.\n");
```